lib/themes/dosomething/paraneue_dosomething/js/campaign/HotModule.js

```diff
@@ -1,3 +1,13 @@
+/* eslint-disable */
```

Comment:
Do we have to disable eslint on any script that isn't written in ES6 / will that mean that these scripts won't be linted at all?